src/braket/devices/devices.py

```diff
@@ -58,7 +58,7 @@ class _Xanadu(str, Enum):
     # DWave = _DWave
     IonQ = _IonQ
     IQM = _IQM
-    OQC = _OQC
+    # OQC = _OQC
```

Comment:
Intentionally commenting this instead of removing this from the Enums?

Reply:
Just doing the same thing we've done with D-Wave and Xanadu